doc/md_src_files/050_decision-points_1.md

```diff
@@ -35,7 +55,21 @@ Therefore, if there is a vulnerability then there must be some technical impact.
 | Partial | The exploit gives the adversary *limited* control over, or information exposure about, the behavior of the software that contains the vulnerability. Or the exploit gives the adversary an importantly low stochastic opportunity for total control. In this context, “low” means that the attacker cannot reasonably make enough attempts to overcome the low chance of each attempt not working. Denial of service is a form of limited control over the behavior of the vulnerable component. |
 | Total   | The exploit gives the adversary *total* control over the behavior of the software, or it gives total disclosure of all information on the system that contains the vulnerability       |
 
-## Utility 
+
+### Gathering Information About Technical Impact
+
+Assessing [*Technical Impact*](#technical-impact) amounts to assessing the degree of control the vulnerability gives an attacker over the vulnerable component.
+One way to approach this analyiss is to ask whether the control gained is *total* or not.
+If it is not total, it is *partial*.
+If an answer to one of the following questions is yes, then control is *total*:
```

Comment:
if you accept the above comment's premise, then the bullets below might need to be prefixed with "after exploiting the vul, does the attacker gain..." or something similar. Point being to highlight the specific change in ability the attacker gains via exploitation (contrast with a possible reading that they have the ability with or without the vulnerability).

Reply:
Added prefix of question before all bullets, for better parallelism.